lib/ansible/plugins/callback/default.py

```diff
@@ -115,6 +115,9 @@ def v2_on_file_diff(self, result):
         if 'diff' in result._result and result._result['diff']:
             self._display.display(self._get_diff(result._result['diff']))
 
+    def _get_censored_item(self, result):
```

Comment:
default is not the correct place for this as other custom callbacks can ignore it then. 
items should be scrubbed before they even get to callbacks, or if not possible, _get_censored_item should be called automatically from the base class (which still leaves v1 callbacks able to ignore it).